webui/react/src/components/DeterminedAuth.tsx

```diff
@@ -101,21 +102,23 @@ const DeterminedAuth: React.FC<Props> = ({ canceler }: Props) => {
         ]}>
         <Input
           autoFocus
+          data-testid="username"
           placeholder="username"
           prefix={<Icon name="user" size="small" title="Username" />}
         />
       </Form.Item>
       <Form.Item name="password">
         <Input.Password
+          data-testid="password"
           placeholder="password"
```

Comment:
Should we just be using the place holder or the accessibility label here? IDK how placeholder is actually used, so might be a bad idea, but it would cut down on double-defining this value.

Reply:
placeholders are vulnerable to being replaced by words in different languages. Accessibility would be a good idea, but it feels weird using that after looking at the experiment table page.